deploy/examples/object-external.yaml

```diff
@@ -16,3 +16,4 @@ spec:
     port: 80
     externalRgwEndpoints:
       - ip: 192.168.39.182
+        # hostname: example.com
```

Comment:
Add the Documentation as well https://rook.io/docs/rook/v1.11/Storage-Configuration/Object-Storage-RGW/object-storage/#connect-to-an-external-object-store

Reply:
Added thanks